### ISSUE TITLE
node-elm-compiler depending on vulnerable package (lodash)

### ISSUE THREAD

**Issue by nullbio** [rtfeldman/node-elm-compiler#92]
A recent vulnerability was found in older versions of `lodash` (the version that `elm-webpack-loader` seems to depend on). Result of `npm audit`:

```
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ elm-webpack-loader [dev]                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ elm-webpack-loader > node-elm-compiler >                     │
│               │ find-elm-dependencies > lodash                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/782                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ elm-webpack-loader [dev]                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ elm-webpack-loader > node-elm-compiler > lodash              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/782                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 2 moderate severity vulnerabilities in 23118 scanned packages
  2 vulnerabilities require manual review. See the full report for details.

```

**Comment by michaelglass** [comment]
fixed in #93

**Comment by jackfranklin** [comment]
Closing this given #93 contains the fix
